packages/click-to-react-component/src/ClickToComponent.js

```diff
@@ -41,10 +41,9 @@ export function ClickToComponent({ editor = 'vscode' }) {
       if (state === State.HOVER && target instanceof HTMLElement) {
         const source = getSourceForElement(target)
         const path = getPathToSource(source)
-        const url = `${editor}://file/${path}`
-
+        const url = `idea://open?file=${path}`
```

Comment:
This looks like a major blocker with a hardcoded value instead of `editor`, isn't it?